vms/proposervm/vm_test.go

```diff
@@ -2530,3 +2531,43 @@ func TestLocalParse(t *testing.T) {
 		})
 	}
 }
+
+func TestTimestampMetrics(t *testing.T) {
+	ctx := context.Background()
+
+	coreVM, _, proVM, _ := initTestProposerVM(t, time.Unix(0, 0), mockable.MaxTime, 0)
+	defer func() {
+		require.NoError(t, proVM.Shutdown(ctx))
+	}()
```

Comment:
```suggestion
	t.Cleanup(func() {
		require.NoError(t, proVM.Shutdown(ctx))
	})
```

Reply:
I'll defer to @StephenButtolph on this due to consistency within the file.

Is there any real difference between the two? If anything I think it would be better to call `t.Cleanup()` inside `initTestProposerVM()` and remove it from everywhere else, but not in this PR.

Reply:
We switched to using `t.Cleanup` in some of our other packages (in holistic passes). Seems like something that could be pulled out into a followup